lib/bip_claims/service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module BipClaims
+  class Service < Common::Client::Base
+    STATSD_KEY_PREFIX = 'api.bip_claims'
+    include Common::Client::Monitoring
+
+    configuration BipClaims::Configuration
+
+    def veteran_attributes(claim)
+      case claim.form_id
+      when '21P-530'
```

Comment:
Is there a case where the form_id will _not_ be 530? What should we do then?

Reply:
This will only be the case if we are trying to use this service for other types of forms beyond 530 burials. For other types of forms we can add logic to accommodate the different key names/data shapes in those forms

Reply:
This is more of a style thing, but Why have a case statement when `if` would do?